packages/system-tests/src/areas/common.ts

```diff
@@ -70,12 +70,12 @@ export class CommonActions {
     );
   }
 
-  public async getConsoleHtml(): Promise<any> {
+  public async getConsoleOutput(): Promise<any> {
```

Comment:
This command was only used to look at cli results on terminal so I'm switching it to return an array of text instead of an array of html that contain the text we are asserting.

Reply:
Yeah, we don't need the html. We just need the text.